test/integration/deployers/blueprints/blueprints_tests.go

```diff
@@ -94,10 +94,7 @@ func DeployerBlueprintTests(f *framework.Framework) {
 			ComponentDescriptorName: "github.com/gardener/landscaper/helm-deployer",
 			BlueprintResourceName:   "helm-deployer-blueprint",
 			DeployItem: func(state *envtest.State, target *lsv1alpha1.Target) (*lsv1alpha1.DeployItem, error) {
-				ref := "eu.gcr.io/gardener-project/landscaper/tutorials/charts/ingress-nginx:4.0.18"
-				if !f.RunOnShoot {
-					ref = "eu.gcr.io/gardener-project/landscaper/tutorials/charts/ingress-nginx:v0.1.0"
-				}
+				ref := "eu.gcr.io/gardener-project/landscaper/examples/charts/hello-world:1.0.1"
```

Comment:
I know that Robert is moving these examples sometime to new locations. Perhaps it would be a good idea to upload the chart to this location https://console.cloud.google.com/gcr/images/gardener-project/eu/landscaper/integration-tests/charts for the integration tests. There is already a simple script you could adapt and use for this: https://github.com/gardener/landscaper-examples/blob/master/hello-world/commands/create-and-push-chart.sh

Reply:
Good point.